controllers/courses.js

```diff
@@ -162,7 +162,10 @@ router.get('/', function(req, res, next) {
                 $or: [
                     { userIds: res.locals.currentUser._id },
                     { teacherIds: res.locals.currentUser._id }
-               ]
+                ],
+                $populate: [
```

Comment:
Loading 16 courses with 25 students each doesn't really effect loading time/performance, <100ms at max, so should be okay